test/sqlite/sqllogic_test_logger.cpp

```diff
@@ -131,7 +135,10 @@ void SQLLogicTestLogger::PrintErrorHeader(const string &file_name, idx_t query_l
 	if (!file_name.empty()) {
 		oss << termcolor::bold << "(" << file_name << ":" << query_line << ")!" << termcolor::reset;
 	}
-	LogFailure(oss.str() + "\n");
+	const char *ci = std::getenv("CI");
+	// check the value is "true" otherwise you'll see the prefix in local run outputs
+	auto prefix = (ci && string(ci) == "true") ? "\n::warning::" : "";
+	LogFailure(prefix + oss.str() + "\n");
```

Comment:
```suggestion
	const char *ci_prepend = std::getenv("CI_PREPEND_TO_ERRORS");
	// check the value is "true" otherwise you'll see the prefix in local run outputs
	auto prefix = ci_prepend ? string("\n") + string(ci_prepend) : string("");
	LogFailure(prefix + oss.str() + "\n");
```
I think it would be somewhat nice to have this be customizable externally, say you prefer this to be `::warning::` or `::error::` or be on a separate infrastructure where you would like to have a different string as identifier.

Also possible should this not sync'd into `LogFailure` ?

Reply:
Thank you for your suggestion, I thought about it yesterday after we discussed it. I like flexibility, when we can change the annotation (maybe even set in json config for unittester), but doesn't adding new env variable mean we must add it into all the workflows? On the other hand, [`CI` env](https://docs.github.com/en/actions/how-tos/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables) comes "for free", so I thought it'd be less noisy to rely on it.

Btw, why should we not sync into `LogFailure`?

Reply:
Customizing can also be done if needed, so it's fine.

Reply:
> I though LogFailure is where this logic should live, so basically moving the first 3 lines, even unchanged, into the implementation of `LogFailure`

`LogFailure` used by many other logger functions so moving those three lines into it would produce a lot of noise in the summary. I've tried it and it looks like this in console output:
```
...

::warning::Wrong column count in query! (test/sql/aggregate/aggregates/histogram_table_function.test:20)!

::warning::Expected 6 columns, but got 2 columns

::warning::================================================================================
SELECT * FROM histogram_values(integers, i, bin_count := 2);

::warning::================================================================================

::warning::The expected result matched the query result.

::warning::Suggested fix: modify header to "query II"

...
```

I'm not sure we want this many warnings. And to include all these lines into one warning, we must create a `::` separated string instead of `\n` separated, which either breaks the output we already have or we'll have to add another buffer for warning message or reimplement failure summary to make it easier to use each failure output as a warning message.

Reply:
Maybe this can be a separate method (`LogFailureHeader`)?